probe/endpoint/ebpf.go

```diff
@@ -125,6 +133,99 @@ func lostCb(count uint64) {
 	ebpfTracker.stop()
 }
 
+func tupleFromPidFd(pid int, fd int) (tuple fourTuple, netns string, ok bool) {
```

Comment:
Can you factor out the common parts for this function with the proc walker?
* Reading net NS
* Reading tcp and tcp6 files